docs/changelog/108571.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 108571
+summary: Workaround G1 bug for JDK 22 and 22.0.1
+area: Infra/CLI
+type: bug
```

Comment:
Should this have a bit of info in here for the release notes?

Reply:
This is about the extent the changelog supports for details. The linked PR (this PR) has more details in the description.